UIKitDZ/UIKitDZ/PhotoFeed/Cells/PostCell/PostTableViewCell.swift

```diff
@@ -0,0 +1,219 @@
+// PostTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Ячейка с опубликовонном постом
+final class PostTableViewCell: UITableViewCell {
+    // MARK: - Visual Components
+
+    private let avatarImageView = UIImageView()
+    private let nameUserLabel = UILabel()
+
+    private let postImageView = UIImageView()
+    private let secondPageImageView = UIImageView()
+    private let thirdPageImageView = UIImageView()
+
+    private let ellipsesLabel = UILabel()
+    private let likeLabel = UILabel()
+    private let dicriptionLabel = UILabel()
+    private let commentedLabel = UILabel()
+    private let publicationTimeLabel = UILabel()
+
+    private let avatarUserImageView = UIImageView()
+    private let likeImageView = UIImageView()
+    private let commentImageView = UIImageView()
+    private let sendImageView = UIImageView()
+    private let saveImageView = UIImageView()
+
+    private let pageControll = UIPageControl()
+    private let scrollView = UIScrollView()
+
+    private lazy var picturePages = [postImageView, secondPageImageView, thirdPageImageView]
+
+    // MARK: - Initializers
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        configureAvatar()
+        configureNickName()
+        configureScroll()
+        setupActionImage()
+        configureLikeLabel()
+        configureDiscription()
+        configureAvatarUser()
+        configureCommentLabel()
+        confugurePublicationTime()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Public Methods
+
+    func configureCell(model: Post) {
+        avatarImageView.image = UIImage(named: model.avatarImage)
```

Comment:
```suggestion
        avatarImageView.image = UIImage(named: model.avatarImage)
```
в этом configureCell все в методы можно разнести